src/Footer/Footer.scss

```diff
@@ -3,6 +3,7 @@
     position: fixed;
```

Comment:
Actually - does this need to be fixed?

I think I'd normally expect to only see the footer if I scrolled right to the bottom of the page.